### DISCUSSION TITLE
可以增加字体大小，显示大小设置选项和字体样式设置吗？

### DISCUSSION THREAD

**Discussion by Plain-Second** [Meloong-Git/PCL#3076] [提问] [answered]
如果电脑设置的显示大小比较小，会让PCL的字看起来很小很费劲，但是如果可以增加一个字体选项，就可以在显示界面小的情况下，字体看起来比较合适，会很舒服
![屏幕截图 2024-01-05 100638](https://github.com/Hex-Dragon/PCL2/assets/150232138/931be00b-5bf7-4a1d-97a0-0f5055831ee3)
![屏幕截图 2024-01-05 100655](https://github.com/Hex-Dragon/PCL2/assets/150232138/321253c0-6a00-41c9-95d2-2df5a9a273d3)

**Comment by WForst-Breeze** [comment, marked as answer]
#1724 等 Issues 提及该内容，但已被拒绝
